meilisearch/client.py

```diff
@@ -205,6 +207,30 @@ def index(self, uid: str) -> Index:
             return Index(self.config, uid=uid)
         raise ValueError("The index UID should not be None")
 
+    def multi_search(self, queries: List[Dict[str, Any]]) -> Dict[str, List[Dict[str, Any]]]:
+        """Multi-index search.
+
+        Parameters
+        ----------
+        queries:
+            List of dictionaries containing the specified indexes and their search queries
+            https://docs.meilisearch.com/reference/api/search.html#search-in-an-index
+
+        Returns
+        -------
+        results:
+            Dictionary of results for each index
```

Comment:
```suggestion
            Dictionary of results for each search query
```

You can have multiple queries on the same indexUid when performing a multi_search